tests/testproject/TestProject.AppHost/TestProgram.cs

```diff
@@ -38,50 +38,32 @@ private TestProgram(string[] args, Assembly assembly, bool includeIntegrationSer
             var mongoDbName = "mymongodb";
             var oracleDbName = "freepdb1";
 
-            var sqlserverContainer = AppBuilder.AddSqlServerContainer("sqlservercontainer")
+            var sqlserver = AppBuilder.AddSqlServer("sqlserver")
                 .AddDatabase(sqlserverDbName);
-            var mysqlContainer = AppBuilder.AddMySqlContainer("mysqlcontainer")
+            var mysql = AppBuilder.AddMySql("mysql")
                 .WithEnvironment("MYSQL_DATABASE", mysqlDbName)
                 .AddDatabase(mysqlDbName);
-            var redisContainer = AppBuilder.AddRedisContainer("rediscontainer");
-            var postgresContainer = AppBuilder.AddPostgresContainer("postgrescontainer")
+            var redis = AppBuilder.AddRedis("redis");
+            var postgres = AppBuilder.AddPostgres("postgres")
                 .WithEnvironment("POSTGRES_DB", postgresDbName)
                 .AddDatabase(postgresDbName);
-            var rabbitmqContainer = AppBuilder.AddRabbitMQContainer("rabbitmqcontainer");
-            var mongodbContainer = AppBuilder.AddMongoDBContainer("mongodbcontainer")
+            var rabbitmq = AppBuilder.AddRabbitMQContainer("rabbitmq");
+            var mongodb = AppBuilder.AddMongoDBContainer("mongodb")
```

Comment:
Did you mean to use the non-container versions of these?

Also, https://github.com/dotnet/aspire/blob/5043b31f5e4a50cf489ccb60c129d3edb1b42c0b/src/Aspire.Hosting/MongoDB/MongoDBBuilderExtensions.cs#L39-L47

.. the code comment says that `AddMongoDb` will use a container for local development. Would it still be using a container then?

(unrelated: how does it decide that this is "local development"?)

Reply:
This is reparation for a change we have coming in P4 where we are consolidating the XYZResource and XYZContainerResource types and extension methods. They both work for local development, the difference is intent when it comes deployment time. But in P4 we'll be introucing another mechanism to specify that intent.

Reply:
I've updated these to use the non-container versions. Nice catch.